README.md

```diff
@@ -1,25 +1,233 @@
 [![codecov](https://codecov.io/gh/chanzuckerberg/cell-census/branch/main/graph/badge.svg?token=byX1pyDlc9)](https://codecov.io/gh/chanzuckerberg/cell-census)
 
-# CELLxGENE Cell Census
+# Cell Census of CZ CELLxGENE Discover
 
-**Status**: Unstable, under rapid development
+[**CZ CELLxGENE Discover**](https://cellxgene.cziscience.com/)  is a free-to-use data portal hosting a growing corpus of more than **700 single-cell datasets** comprising about **50 million cells** from the major human and mouse tissues. The portal provides a set of visual tools to download and explore the data. **All data is [standardized](https://github.com/chanzuckerberg/single-cell-curation/tree/main/schema/3.0.0)** to include raw counts and a common vocabulary for gene and cell metadata.
 
-This repository contains documentation and example code related to the Chan Zuckerberg CELLxGENE Cell Census, and a client (API) package to simplify accessing the Cell Census data.
+The **Cell Census** provides efficient computational tooling to access, query, and analyze all RNA data from CZ CELLxGENE Discover. Using a **new access paradigm of cell-based slicing and querying**, you can interact with the data through [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA), or obtain slices of the data in [AnnData](https://anndata.readthedocs.io/) or [Seurat](https://satijalab.org/seurat/) objects.
 
-The CZ Cell Census is an aggregation of all public single cell data available in [CELLxGENE Discover](https://cellxgene.cziscience.com/), published in API-accessible formats, including the [SOMA API](https://github.com/single-cell-data/).
 
-### Reuse
+Get started on using the Cell Census:
 
-The contents of this Github repository are freely available for reuse under the [MIT license](https://opensource.org/licenses/MIT). Data in the CZ Cell Census are available for re-use under the [CC-BY license](https://creativecommons.org/licenses/by/4.0/).
+- [Quick start](#Quick-start).
+- [Documentation](https://chanzuckerberg.github.io/cell-census/index.html). *Under development.*
+- [Python tutorials](https://github.com/chanzuckerberg/cell-census/tree/main/api/python/notebooks).
+- R tutorials. *Coming soon.*
 
-### For More Help
+## Motivation: Single-cell analysis at scale 
```

Comment:
I suggest removing this section. I believe it is covered with superior prose in the introduction.